cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/node/listener/PlayerManagerListener.java

```diff
@@ -62,15 +62,15 @@ public void handle(ChannelMessageReceiveEvent event) {
             case "update_offline_cloud_player": {
                 ICloudOfflinePlayer cloudOfflinePlayer = event.getData().get("offlineCloudPlayer", CloudOfflinePlayer.TYPE);
 
-                if (cloudOfflinePlayer != null) {
+                if (cloudOfflinePlayer != null) { //todo shouldn't we call the BridgeUpdateCloudOfflinePlayerEvent here? and if this message comes from a service (and not from a node), call updateOfflinePlayer instead of updateOfflinePlayer0
```

Comment:
The event gets called in the BridgeCustomChannelMessageListener

Reply:
Removed